src/pybind/mgr/ssh/module.py

```diff
@@ -249,6 +249,9 @@ def _require_hosts(self, hosts):
                 ", ".join(map(lambda h: "'{}'".format(h),
                     unregistered_hosts))))
 
+    def _get_random_host(self):
+        return random.choice(list(self.inventory_cache.keys()))
+
```

Comment:
Comment by @shaas regarding choosing a random host for scheduling daemons:

> Random does not make sense to me...
Either use "Spread", "Stack" or "Load" as low-level scheduling mechanism.
> The thing is that you need to know the state of your cluster... I am not sure if the ssh-orchestrator has all the needed information to make such decisions.